### ISSUE TITLE
Remove secondary_logo field from SiteConfig and Related Code ✂️ 

### ISSUE THREAD

**Issue by mstruve** [forem/forem#13586]
## Summary

Admins are a bit confused when it comes to providing a secondary_logo. We already have a field for providing logo and this one is "secondary" which means it's basically an alternative version of the logo. But admins can't really tell WHY we ask them to provide it and WHERE it would be used. Communities often hardly have one logo, not even mention a secondary one :). For this reason we are removing the secondary_logo as a configurable option and replacing the current use places for it with the regular logo. 

- [ ] Remove secondary_logo field and code
- [ ] Replace all places that use the secondary_logo with main logo (logo_png or logo_svg)
- [ ] Communicate this change via the changelog. 

## Additional Context

It is "inspired" by the the rough [roll out plan for Starter Pack](https://docs.google.com/document/d/1facy0-UXlUPrzpll8TAM-MNjZiwVbO1vepphuE6a-ek/edit).
Product approved stage 0: https://github.com/forem/rfcs/issues/208

**Comment by citizen428** [comment]
I assigned this to myself because I needed a break from my other `SiteConfig` work... Anyway, any idea what's going to happen to `logo_svg`? The only place we really seem to be using this is `app/views/layouts/_logo.html.erb` and I wonder if we can just get rid of that one too.

**Comment by ludwiczakpawel** [comment]
@citizen428 

[Here's](https://github.com/forem/rfcs/blob/a8d843d9a37c4105850f575f6e51faac7377a474/text/0190-admin-onboarding.md#fields) (see point 2, titled "Logo") what's gonna happen (at some point soon hopefully) with logo_svg and generally all logo assets we provide currently - I think it provides pretty good explanation.

TL;DR: we want to unify all logo-related fields into one field which will be `input[type="file"]` field supporting SVG, JPG, JPEG & PNG formats.

**Comment by citizen428** [comment]
Awesome, thanks @ludwiczakpawel!
